app/routes.py

```diff
@@ -25,3 +25,11 @@ def add_item():
     item = request.get_json()
     items.append(item)
     return {'message': 'Item added successfully'}, 201
```

Comment:
I just realised that the user won't know the ID of their item, which makes it hard to delete (or get!) it. Can you return the item ID here?

Reply:
Zero-indexed IDs are a bit nasty. Can you start from 1?